notesapi/v1/migrations/0001_initial.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
```

Comment:
Don't get me talking about it... Friday was a frustrating day. This project is using Django 1.7, which handles migrations differently.